nix/treefmt.nix

```diff
@@ -24,13 +24,13 @@
         # Python
         # It was found out that the best outcome comes from running mulitple
         # formatters.
-        black.enable = true; # The Classic Python formatter
         isort.enable = true; # Python import sorter
         # Ruff, a Python formatter written in Rust (30x faster than Black).
         # Also provides additional linting.
         # Do not enable ruff.format = true; because then it won't complaing
```

Comment:
Ah I didn't read this. I don't see any linting errors complaints though? So maybe this has been fixed?